### PR TITLE
refractored Charsets to StandardCharsets Fixes #7884

### LINKED ISSUE #7884: Replace com.google.common.base.Charsets with java.nio.charset.StandardCharsets in EtcdRepository.java
Same as #7883 

- `org.apache.shardingsphere.governance.repository.etcd.EtcdRepository`
- `org.apache.shardingsphere.governance.repository.etcd.EtcdRepositoryTest`

### PR DESCRIPTION
**Pull request by TanveshT**
Fixes #7884.

Changes proposed in this pull request:
- refactoring and import statements changed to the expected class names

### CODE REVIEW

**Comment by menghaoranss** [comment]
@TanveshT thanks for your pr, please fix the checkstyle error:
```
Error:  src/main/java/org/apache/shardingsphere/governance/repository/etcd/EtcdRepository.java:[112] (sizes) LineLength: Line is longer than 200 characters (found 215).
```

**Comment by TanveshT** [comment]
@menghaoranss Okay I will fix it asap

**Comment by menghaoranss** [comment]
> @menghaoranss Okay I will fix it asap
> I wanted to ask that what is the checkstyle constraints? like if I just move a .function to the next line will it trigger some other checkstyle warning?

`,` must be on the first line.

**Comment by menghaoranss** [comment]
@TanveshT any problems?

**Comment by TanveshT** [comment]
Hey I'll first install check style on my local system and then commit asap

**Comment by menghaoranss** [comment]
> Hey I'll first install check style on my local system and then commit asap

Okay, this is the checkstyle rule for shardingsphere :
`https://github.com/apache/shardingsphere/blob/master/src/resources/checkstyle.xml`

**Comment by TanveshT** [comment]
@menghaoranss While loading the checkstyle in intellij, it's throwing this error can you help?
```bash
com.puppycrawl.tools.checkstyle.api.CheckstyleException: unable to parse configuration stream - Element type "html" must be declared.:7:17
	at com.puppycrawl.tools.checkstyle.ConfigurationLoader.loadConfiguration(ConfigurationLoader.java:322)
	at com.puppycrawl.tools.checkstyle.ConfigurationLoader.loadConfiguration(ConfigurationLoader.java:287)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.infernus.idea.checkstyle.service.cmd.OpLoadConfiguration.loadConfigurationForCheckstyle825AndAbove(OpLoadConfiguration.java:188)
	at org.infernus.idea.checkstyle.service.cmd.OpLoadConfiguration.callLoadConfiguration(OpLoadConfiguration.java:156)
	at org.infernus.idea.checkstyle.service.cmd.OpLoadConfiguration.execute(OpLoadConfiguration.java:139)
	at org.infernus.idea.checkstyle.service.cmd.OpCreateChecker.loadConfig(OpCreateChecker.java:98)
	at org.infernus.idea.checkstyle.service.cmd.OpCreateChecker.execute(OpCreateChecker.java:54)
	at org.infernus.idea.checkstyle.service.cmd.OpCreateChecker.execute(OpCreateChecker.java:26)
	at org.infernus.idea.checkstyle.service.CheckstyleActionsImpl.executeCommand(CheckstyleActionsImpl.java:130)
	at org.infernus.idea.checkstyle.service.CheckstyleActionsImpl.createChecker(CheckstyleActionsImpl.java:60)
	at org.infernus.idea.checkstyle.service.CheckstyleActionsImpl.createChecker(CheckstyleActionsImpl.java:51)
	at org.infernus.idea.checkstyle.checker.CheckerFactoryWorker.run(CheckerFactoryWorker.java:46)
Caused by: org.xml.sax.SAXParseException; lineNumber: 7; columnNumber: 17; Element type "html" must be declared.
	at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source)
	at org.apache.xerces.util.ErrorHandlerWrapper.error(Unknown Source)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source)
	at org.apache.xerces.impl.dtd.XMLDTDValidator.handleStartElement(Unknown Source)
	at org.apache.xerces.impl.dtd.XMLDTDValidator.startElement(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanStartElement(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentScannerImpl$ContentDispatcher.scanRootElementHook(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source)
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.puppycrawl.tools.checkstyle.XmlLoader.parseInputSource(XmlLoader.java:86)
	at com.puppycrawl.tools.checkstyle.ConfigurationLoader.parseInputSource(ConfigurationLoader.java:198)
	at com.puppycrawl.tools.checkstyle.ConfigurationLoader.loadConfiguration(ConfigurationLoader.java:315)
	... 15 more
```

**Comment by kimmking** [comment]
@TanveshT  this xml file is used by maven, not IDEA.

File: shardingsphere-governance/shardingsphere-governance-repository/shardingsphere-governance-repository-provider/shardingsphere-governance-repository-etcd/src/main/java/org/apache/shardingsphere/governance/repository/etcd/EtcdRepository.java [hunk 1]
```diff
@@ -109,11 +109,12 @@ public void watch(final String key, final DataChangedEventListener dataChangedEv
             for (WatchEvent each : response.getEvents()) {
                 ChangedType changedType = getEventChangedType(each);
                 if (ChangedType.IGNORED != changedType) {
-                    dataChangedEventListener.onChange(new DataChangedEvent(each.getKeyValue().getKey().toString(Charsets.UTF_8), each.getKeyValue().getValue().toString(Charsets.UTF_8), changedType));
+                    dataChangedEventListener.onChange(new DataChangedEvent(each.getKeyValue().getKey().toString(StandardCharsets.UTF_8)
+                            , each.getKeyValue().getValue().toString(StandardCharsets.UTF_8), changedType));
```
Comment (menghaoranss) [inline shardingsphere-governance/shardingsphere-governance-repository/shardingsphere-governance-repository-provider/shardingsphere-governance-repository-etcd/src/main/java/org/apache/shardingsphere/governance/repository/etcd/EtcdRepository.java:113]: `,` must be on the first line.

**Review by menghaoranss — CHANGES_REQUESTED** [review]

File: shardingsphere-governance/shardingsphere-governance-repository/shardingsphere-governance-repository-provider/shardingsphere-governance-repository-etcd/src/main/java/org/apache/shardingsphere/governance/repository/etcd/EtcdRepository.java (hunk 1 above)
Comment (TanveshT) [reply shardingsphere-governance/shardingsphere-governance-repository/shardingsphere-governance-repository-provider/shardingsphere-governance-repository-etcd/src/main/java/org/apache/shardingsphere/governance/repository/etcd/EtcdRepository.java:113]: @menghaoranss first line means the same indent?

Comment (menghaoranss) [reply shardingsphere-governance/shardingsphere-governance-repository/shardingsphere-governance-repository-provider/shardingsphere-governance-repository-etcd/src/main/java/org/apache/shardingsphere/governance/repository/etcd/EtcdRepository.java:113]:
```
dataChangedEventListener.onChange(new DataChangedEvent(each.getKeyValue().getKey().toString(StandardCharsets.UTF_8),
                            each.getKeyValue().getValue().toString(StandardCharsets.UTF_8), changedType));
```

Comment (menghaoranss) [reply shardingsphere-governance/shardingsphere-governance-repository/shardingsphere-governance-repository-provider/shardingsphere-governance-repository-etcd/src/main/java/org/apache/shardingsphere/governance/repository/etcd/EtcdRepository.java:113]: @TanveshT Hi, any other questions?

Comment (TanveshT) [reply shardingsphere-governance/shardingsphere-governance-repository/shardingsphere-governance-repository-provider/shardingsphere-governance-repository-etcd/src/main/java/org/apache/shardingsphere/governance/repository/etcd/EtcdRepository.java:113]: Nothing else @menghaoranss. Will make the that commit soon

**Review by menghaoranss — APPROVED** [review]

**Pull request merged**